src/wagtailmedia/wagtail_hooks.py

```diff
@@ -109,7 +109,7 @@ class MediaAdminURLFinder(ModelAdminURLFinder):
 register_admin_url_finder(get_media_model(), MediaAdminURLFinder)
 
 
-@hooks.register("insert_editor_css")
+@hooks.register("insert_global_admin_css")
```

Comment:
while I would've preferred a single hook implementation, this is good enough.

Note to self: got to do more clean up, and potentially drop this CSS file entirely - 
https://github.com/torchbox/wagtailmedia/blob/main/src/wagtailmedia/static/wagtailmedia/css/wagtailmedia.css#L5-L13